docker/flags.go

```diff
@@ -107,6 +107,7 @@ func init() {
 			{"run", "Run a command in a new container"},
 			{"save", "Save an image to a tar archive"},
 			{"search", "Search for an image on the Docker Hub"},
+			{"set", "Setup or update resource configs for one or more containers"},
```

Comment:
`Update properties of one or more running containers`.
If its too long, leave off "running".